MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TabControl.xaml

```diff
@@ -199,59 +198,73 @@
+        <Setter Property="Background" Value="Transparent"/>
         <Setter Property="Foreground" Value="{DynamicResource MaterialDesignBody}"/>
-        <Setter Property="Padding" Value="16"/>
+        <Setter Property="Padding" Value="0"/>
         <Setter Property="Height" Value="72" />
         <Setter Property="Width" Value="72" />
+        <Setter Property="Margin" Value="4"/>
         <Setter Property="wpf:RippleAssist.Feedback" Value="{DynamicResource MaterialDesignFlatButtonRipple}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type TabItem}">
-                    <Grid x:Name="Root" >
-                        <wpf:ColorZone x:Name="tabitemcz"
-                                       HorizontalAlignment="Stretch"
-                                       VerticalAlignment="Stretch"
-                                       Focusable="False"
-                                       Mode="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Mode)}" 
-                                       Background="{x:Null}"
-                                       wpf:ColorZoneAssist.Background="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Background)}"
-                                       wpf:ColorZoneAssist.Foreground="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(wpf:ColorZoneAssist.Foreground)}">
-                            <wpf:Ripple Focusable="False"
-                                        ClipToBounds="True"
-                                        Content="{TemplateBinding Header}" 
-                                        ContentTemplate="{TemplateBinding HeaderTemplate}"
-                                        ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
-                                        ContentStringFormat="{TemplateBinding HeaderStringFormat}"
-                                        HorizontalContentAlignment="Center"
-                                        VerticalContentAlignment="Center"
-                                        RecognizesAccessKey="True"
-                                        x:Name="contentPresenter"
-                                        Opacity=".82"
-                                        SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" 
-                                        Padding="{TemplateBinding Padding}"
-                                        TextBlock.FontWeight="Medium"
-                                        TextBlock.FontSize="15"
-                                        TextOptions.TextFormattingMode="Ideal" 
-                                        TextOptions.TextRenderingMode="Auto">
-                            </wpf:Ripple>
-                        </wpf:ColorZone>
+                    <Grid x:Name="Root"
+                          Cursor="Hand">
```

Comment:
Putting it in the setter results in the cursor changing shape both when over the button **and** when over the tab content. Like this:
![MaterialDesignDemo_MWAyQUCjsu](https://user-images.githubusercontent.com/58171461/122175252-c836bc00-ce83-11eb-9cd3-f2d38d3b8aa8.png)

It could be solved by making a custom property, but I'm not sure it's worth it. What do you think?

Reply:
That is a good point. I think for now, lets omit setting the cursor bits explicitly.